Packs/DragosWorldview/Integrations/DragosWorldview/DragosWorldview.py

```diff
@@ -1,5 +1,7 @@
 import demistomock as demisto  # noqa: F401
 from CommonServerPython import *  # noqa: F401
+
+
```

Comment:
@NicCas remove these empty lines, just to remove the diff

Reply:
Completed